README.md

```diff
@@ -981,9 +982,46 @@ Optional configuration:
         onFailure(error);
      }
     ```
-## OpenId Connect
 
-Options for the [OpenId Connect](http://developer.okta.com/docs/api/resources/oidc.html) authentication flow. This flow is required for social authentication, and requires OAuth client registration with Okta. For instructions, see [Social Authentication](http://developer.okta.com/docs/api/resources/social_authentication.html).
+## IdP Discovery
+**EA feature:** The Idp Discovery feature is currently a [EA feature](https://developer.okta.com/docs/api/getting_started/releases-at-okta#early-access-ea).
```

Comment:
nitpick: The **IdP** Discovery feature is currently **an** EA feature

Can you format it like this?

> :information_source: The IdP Discovery feature is currently an [EA feature](https://developer.okta.com/docs/api/getting_started/releases-at-okta#early-access-ea).